_upgrade-to/upgrade-to.md

```diff
@@ -87,7 +87,7 @@ If you are migrating an Open Distro for Elasticsearch cluster, we recommend firs
    # Elasticsearch OSS
    curl -XGET 'localhost:9200/_nodes/_all?pretty=true'
    # Open Distro for Elasticsearch with Security plugin enabled
-   curl -XGET 'https://localhost:9200/_nodes/_all?pretty=true' -u 'admin:admin' -k
+   curl -XGET 'https://localhost:9200/_nodes/_all?pretty=true' -u 'admin:< Admin password >' -k
```

Comment:
```suggestion
   curl -XGET 'https://localhost:9200/_nodes/_all?pretty=true' -u 'admin:<custom-admin-password>' -k
```

Reply:
Applied suggestion in: [cdeb90f](https://github.com/opensearch-project/documentation-website/pull/5887/commits/cdeb90ffae8be7a3ae2f9de267abc08879dd44cd)